index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>PORTFOLIO MICROVERSE</title>
+    <script src="https://kit.fontawesome.com/de9dd03ce2.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <main>
+        <header>
+            <h2 class="my-logo">Aneal Laryea</h2>
+            <i class="fa-solid fa-bars"></i>
+        </header>
+    
+        <section class="headline">
+            <div class="primary-text">
+                <h3>Hey There.
+                    I’m Aneal</h3>
```

Comment:
- Please use the tag `<h1>` here
-  Please use the proper font size as in the Figma `56px` yours is `65.52px`
- Please use the proper font-family `Crete Round`

##### Your design
![Screenshot from 2022-05-26 18-07-45](https://user-images.githubusercontent.com/58771507/170539117-12175295-d192-4aed-a762-fa8c7a45868f.png)

##### Figma design
![Screenshot from 2022-05-26 18-08-34](https://user-images.githubusercontent.com/58771507/170539249-8daeb2a1-a174-4ec9-86b9-330bc897edd8.png)